web/yo/app/scripts/services/stringutils.js

```diff
@@ -1091,7 +1091,7 @@ angular.module('oncokbApp')
                 if (model.get(e)) {
                     if (model.get(e).type === 'Map') {
                         reviewObj[e] = getReview(model[e]);
-                    } else {
+                    } else if(_.isString(model.get(e))) {
                         reviewObj[e] = getString(model.get(e));
```

Comment:
what is CollectiveString? 
If you mean collaborative string, then, we don't need to save it so far. Currently only currentReviewer is in collaborative string, which we don't need to save. Since it is only used for review process.